app/controllers/base_controller.rb

```diff
@@ -34,4 +36,9 @@ def set_order_cycles
       current_order(true).set_order_cycle! @order_cycles.first
     end
   end
+
+  def ensure_shop_ready
+    # Don't display order cycles if shop is not ready for checkout
+    @order_cycles = {} unless @distributor.ready_for_checkout?
```

Comment:
It doesn't seem wise to me to first query order cycles and then just forget about them. We can check that pre-condition first, right?